Vagrantfile

```diff
@@ -1,9 +1,9 @@
 # -*- mode: ruby -*-
 
 MACHINES = {
-  publicdb: {hostname: "publicdb.localdomain", ip: "192.168.99.11", ssh: 2021, http: 8081},
-  vpn: {hostname: "vpn.localdomain", ip: "192.168.99.12", ssh: 2022, http: 8082},
-  datastore: {hostname: "datastore.localdomain", ip: "192.168.99.13", ssh: 2023, http: 8083}
```

Comment:
You should use a different mapped port for each machine, otherwise if you are running multiple locally you will get conflicts since they will all try to map their 443 to your localhost:8443.

Also missing space after the colons.

Reply:
Ouch! Will do.